packages/react-magma-dom/src/components/TreeView/TreeItem.tsx

```diff
@@ -334,19 +515,21 @@ export const TreeItem = React.forwardRef<HTMLLIElement, TreeItemProps>(
           selected={selectedItem}
           theme={theme}
           tabIndex={tabIndex}
-          onKeyDown={handleKeyDown}
+          onKeyDown={onKeyDownHandler}
+          ref={treeItemRef}
```

Comment:
Oooh! It looks like this may help with this new bug I just opened, yay https://github.com/cengage/react-magma/issues/1800